app/commands/solution/publish.rb

```diff
@@ -13,7 +13,6 @@ def call
         ActiveRecord::Base.transaction do
           solution.update(published_at: Time.current)
           Solution::PublishIteration.(solution, iteration_idx)
-          Solution::UpdateSnippet.(solution)
```

Comment:
I've removed this as `Solution::UpdateSnippet` is now also called from `Solution::PublishIteration` (as per https://github.com/exercism/website/pull/2712).